server/controllers/valuesController.js

```diff
@@ -1,11 +1,55 @@
 const { db } = require('../dbConfig');
 
 const valuesController = {
+  getAllValues: async () => {
+    return await db('company_values')
+      .select()
+      .then((values) => createRes(values));
+  },
   updateValue: async (companyValue) => {
     return await db('company_values')
       .where({ value: companyValue.value })
       .update({ ...companyValue }, ['id']);
   },
 };
 
+const createRes = async (values) => {
+  let results = [];
+
+  for (let i = 0; i < values.length; i++) {
+    const num_times_used = await db('shoutouts')
+      .where('company_value', values[i].id)
+      .count();
+    const top_shoutee = await getTopUser(values[i].id, 'shoutee');
+    const top_shouter = await getTopUser(values[i].id, 'shouter');
+
+    results.push({
+      active: values[i].active_status,
+      description: values[i].description,
+      num_times_used: num_times_used[0].count,
```

Comment:
Why snakecase instead of camelcase?

Reply:
From everything I've seen, server side is usually snake case while front end is camel case. I'm not sure that there's a rule, but I like the distinction personally